src/utilities/element-utilities-extender-factory.js

```diff
@@ -1025,17 +1030,27 @@ module.exports = function () {
       cy.startBatch();
 
```

Comment:
At the beginning of this function 'createTemplateReaction', let's change the comment 
"complexName: The name of the complex in the reaction." as
"complexName: The name of the complex in the reaction. However, in template 'reversible', this parameter represents the macromolecules in the right hand-side of the reaction. These macromolecules are also referred as inputMacromolecules inside the function."